__tests__/server.test.js

```diff
@@ -434,4 +434,29 @@ describe("API endpoints", () => {
                 });
         });
     });
+    describe("DELETE: /api/comments/:comment_id", () => {
+        test("204 - deletes a comment when provided with a comment id", () => {
+            return request(app).delete("/api/comments/7").expect(204);
+        });
+        test("400 - sends an appropriate error message when the client tries to delete a comment using an invalid comment id", () => {
+            return request(app)
+                .delete("/api/comments/invalid-id")
+                .expect(400)
+                .then((response) => {
+                    expect(response.body.msg).toBe(
+                        "ERROR: bad request - invalid input"
+                    );
+                });
+        });
+        test("404 - sends an appropriate error message when the client tries to delete a comment using a comment id that does not exist", () => {
+            return request(app)
+                .delete("/api/comments/752")
```

Comment:
You could store 752 in a variable and give it some semantic meaning. Something like `const NON_EXISTANT_ID = 752` and then use a template string. This would make it clear why 752 is important